frontend/SideDish/SideDish/View/HeaderView.swift

```diff
@@ -0,0 +1,17 @@
+
+import UIKit
+
+class HeaderView: UICollectionReusableView {
+
+    @IBOutlet weak var headerButton: UIButton!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+    
+    func configureButtonName(text: String) {
+        self.headerButton.setTitle(text, for: .normal)
+    }
+    @IBAction func buttonPressed(_ sender: Any) {
```

Comment:
sender 타입도 이벤트 발생하는 타입으로 작성할 수 있습니다. HeaderView 내부에서 사용하는 로직이라면 private 접근제어자 명시하는 것이 좋습니다!

```suggestion
    @IBAction private func buttonPressed(_ sender: UIButton) {
```

Reply:
앞으로 내부에서 사용하는 메소드도 접근제어자를 명시하도록 하겠습니다!